app/src/main/java/com/example/keniphhomemaintenance/Navigation.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.keniphhomemaintenance
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+import com.example.keniphhomemaintenance.dwellings.DwellingsScreenView
+import com.example.keniphhomemaintenance.ui.Screen
+
+@ExperimentalComposeUiApi
+@Composable
+fun Navigation() {
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = Screen.DwellingsScreen.route) {
+        composable(route = Screen.DwellingsScreen.route) {
+            DwellingsScreenView(navController = navController)
+        }
+        composable(
+            route = Screen.UtilitiesScreen.route + "/{name}" + "/{address}",
+            arguments = listOf(
```

Comment:
Passing name and address to the next screen as placeholders.
In the future we'd want to store the data in a way that the utilities/items to be repaired stays tied to the respective dwelling.
Each `Dwelling` object has an `id` attribute we can potentially use, but unsure if passing `id` through navigation is the best way to keep these connected.